geom/geom/inc/TGeoPhysicalConstants.h

```diff
@@ -0,0 +1,133 @@
+// -*- C++ -*-
```

Comment:
Are these headers from CLHEP? They are missing the ROOT copyright/licensing terms, mentioning of course the source.

Reply:
There is a more general question: I imagine other parts of ROOT needing system of units and physical constants and it would make sense to add these 2 headers in libCore. @peremato what is your take?

Reply:
We discussed this issue in private, we will merge as it is and figure out later how to deal with this more generally